lib/senkyoshi.rb

```diff
@@ -11,6 +11,8 @@
 require "nokogiri"
 require "zip"
 
+Zip.write_zip64_support = true
```

Comment:
Just my opinion, but I feel this is not in the scope of this PR and should be its own PR.